R/infix_spaces_linter.R

```diff
@@ -112,6 +112,22 @@ infix_spaces_linter <- function(exclude_operators = NULL, allow_multiple_spaces
 
     bad_expr <- xml2::xml_find_all(xml, xpath)
 
+    # Filter `box::use()` declarations
+    bad_expr <- Filter(
+      function (expr) {
+        is.na(xml2::xml_find_first(
+          expr,
+          "self::OP-SLASH[
+            ancestor::expr/preceding-sibling::OP-LEFT-PAREN/preceding-sibling::expr[
+              ./SYMBOL_PACKAGE[text() = 'box'] and
+              ./SYMBOL_FUNCTION_CALL[text() = 'use']
+            ]
+          ]"
```

Comment:
The XPath looks sound.
I think it would be preferrable to include it as a `not()` condition in the bad_expr XPath though.
Is there any infix operator that can occur in `box::use()` expressions and that should be linted?
Otherwise I think this can simply be appended to the conditions via `and not(...)`.